File: jeanstationfrontend/src/app/models/Address.ts
export default class Address{
    addressId: string;
    userId: string;
    streetName: string;
    apartmentNumber: string;
    city: string;
    province: string;
    postalcode: string;

    constructor(){
        this.addressId = "";
        this.userId = "";
        this.apartmentNumber = "";
        this.streetName = "";
        this.city = "";
        this.province = "";
        this.postalcode = "";
    }
}

File: NotificationsAPI/Models/Address.cs
namespace NotificationsAPI.Models
{
    public class Address
    {
        public string AddressId { get; set; }
        public string UserId { get; set; }
        public string StreetName { get; set; }
        public string ApartmentNumber { get; set; }
        public string City { get; set; }
        public string Province { get; set; }
        public string PostalCode { get; set; }
    }
}
